cities/frankfurtmain.json

```diff
@@ -0,0 +1,325 @@
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    8.6300325,
+                    50.1578427
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "location": "Walter-Möller-Platz 3",
+                "opening_hours": "We 09:00-18:00",
+                "title": "Wochenmarkt Nordwestzentrum"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    8.6970994,
+                    50.1107928
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "location": "Paul-Arnsberg-Platz ",
+                "opening_hours": "Fr 09:00-18:00",
+                "title": "Wochenmarkt Nordend"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    8.6963383,
+                    50.155387
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "location": "Gravensteiner-Platz",
+                "opening_hours": "Fr 11:00-19:00",
+                "title": "Wochenmarkt Preungesheim"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    8.6278133,
+                    50.177138
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "location": "Riedbergplatz",
+                "opening_hours": "Sa 09:00-16:00",
+                "title": "Wochenmarkt Riedberg"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    8.6835913,
+                    50.10054
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "location": "Diesterwegplatz 50",
+                "opening_hours": "Tu, Fr 08:00-18:00",
+                "title": "Wochenmarkt Sachsenhausen"
+            },
+            "type": "Feature"
+        },
+        {
+            "geometry": {
+                "coordinates": [
+                    8.5437426,
+                    50.0993068
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "location": "Höchster Markt 6",
+                "opening_hours": "Tu, Fr, Sa 7:00 - 13:00",
+                "title": "Wochenmarkt und Markthalle Höchst"
+            },
+            "type": "Feature"
+        }
+
+    ],
+    "metadata": {
+        "data_source": {
+            "title": "Stadt Frankfurt am Main",
+            "url": "http://frankfurt.de/sixcms/detail.php?id=4623&_ffmpar[_id_eltern]=4622"
+        },
+        "map_initialization": {
+            "coordinates": [
```

Comment:
The markets on the map are still not centered. I will adjust this before I merge your branch.